eahub/localgroups/views.py

```diff
@@ -1,14 +1,38 @@
 import json
-from datetime import datetime 
+from datetime import datetime
 
 from django.urls import reverse_lazy
 from django.shortcuts import render, redirect, get_object_or_404
 from django.contrib.auth.decorators import login_required
-
+from django.core.mail import mail_managers
+from django.contrib.sites.shortcuts import get_current_site
+from django.contrib import messages
+from django.template.loader import render_to_string
 from .models import LocalGroup as Group
 
 def view_group(request, slug):
     group = get_object_or_404(Group, slug=slug)
+
     return render(request, 'eahub/group.html', {
         'group': group
     })
+
+@login_required(login_url=reverse_lazy('login'))
+def claim_group(request, slug):
+    group = get_object_or_404(Group, slug=slug)
+    if request.method == 'POST':
+        subject = "EA Group claimed: {0}".format(group.name)
+        user_eahub_url = "https://{0}/profile/{1}".format(get_current_site(request).domain,request.user.profile.slug)
```

Comment:
It'd be good to use [`get_absolute_url`](https://docs.djangoproject.com/en/2.1/ref/models/instances/#django.db.models.Model.get_absolute_url) here instead of manually stitching together a URL.

Also, it's technically possible for a user to not have a profile, and this code will raise a 500 error in that case; you need to handle it. The handling doesn't need to be especially pretty, though, because for now this should be rare.